Dockerfile

```diff
@@ -31,21 +31,20 @@ FROM php-base-${FLAVOUR} AS php-base
 
 # Debian PHP with dependencies needed for building the tools
 FROM php-base-debian AS builder-base-debian
-ARG DEBIAN_BUILD_DEPS="autoconf file g++ gcc libc-dev pkg-config re2c"
 RUN --mount=type=cache,target=/var/cache/apt,target=/var/cache/apt,sharing=locked,id=apt \
     --mount=type=cache,target=/var/lib/apt/lists,target=/var/lib/apt/lists,sharing=locked,id=apt-lists \
-    apt-get install -y --no-install-recommends ${DEBIAN_BUILD_DEPS}
+    apt-get install -y --no-install-recommends ${PHPIZE_DEPS}
```

Comment:
`PHPIZE_DEPS` isn't defined anywhere in our Dockerfile. Is it defined by the parent image?

Reply:
https://github.com/docker-library/php/blob/142e8040a02cd253fac6c09a53a8e58ec4595932/8.1/buster/cli/Dockerfile#L18-L29=

Reply:
Yes they're defined by the official PHP image, which is better if they change